packages/common-lib/src/services/courseRegistryService.js

```diff
@@ -256,6 +256,7 @@ export const moduleTracking = async (
         headers
       }
     )
+    console.log(result.data)
```

Comment:
@Aar-if Please remove this, lot of data coming in console. We will have to check not required consoles and remove those